routes/drones.js

```diff
@@ -1,37 +1,59 @@
 const express = require('express');
 
 // require the Drone model here
-
+const Drone = require('../models/Drone.model')
 const router = express.Router();
 
 router.get('/drones', (req, res, next) => {
-  // Iteration #2: List the drones
-  // ... your code here
+  Drone.find({})
+        .then((drones) => {
+          res.render("drones/list", { drones })
+        })
+        .catch((err) => console.log(`Could not render /drones path: ${err}`))
 });
 
 router.get('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+  res.render("drones/create-form")
 });
 
 router.post('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+  const { name, propellers, maxSpeed } = req.body;
+
+  Drone.create({ name, propellers, maxSpeed })
+    .then(() => res.redirect("/drones"))
+    .catch((err) => console.log(`Could not render the creation of the drone: ${err}`))
 });
 
 router.get('/drones/:id/edit', (req, res, next) => {
-  // Iteration #4: Update the drone
-  // ... your code here
+  const { id } = req.params;
+  console.log(`the id is ${id}`);
```

Comment:
You should remove `console.logs` from your codebase once you've confirmed everything works and they no longer serve a purpose.